### PR TITLE
gtk: Add binding for `gtk_container_class_handle_border_width()`

### PR DESCRIPTION
**Pull request by ids1024**

### CODE REVIEW

File: gtk/src/subclass/container.rs [hunk 1]
```diff
@@ -268,3 +268,19 @@ impl Callback {
...
+
+pub unsafe trait ContainerClassSubclassExt: ClassStruct {
+    #[doc(alias = "gtk_container_class_handle_border_width")]
+    fn handle_border_width(&mut self) {
+        unsafe {
+            let type_class = self as *mut _ as *mut glib::gobject_ffi::GTypeClass;
+            let widget_class = glib::gobject_ffi::g_type_check_class_cast(
+                type_class,
+                ffi::gtk_container_get_type(),
+            ) as *mut ffi::GtkContainerClass;
```
Comment (bilelmoussaoui) [inline gtk/src/subclass/container.rs:280]:
```suggestion
            let widget_class = glib::gobject_ffi::g_type_check_class_cast(
                type_class,
                ffi::gtk_container_get_type(),
            ) as *mut ffi::GtkContainerClass;
```
Is this needed? the check is already implied by implementing the trait only for widgets that implement `ContainerImpl`. I suppose you can directly cast it to as `GtkContainerClass`

Comment (ids1024) [reply gtk/src/subclass/container.rs:280]: I copied it from the methods of `WidgetClassSubclassExt`. But yeah, probably not necessary.

Comment (sdroege) [reply gtk/src/subclass/container.rs:280]: Yes, not necessary and also not there :)

**Review by sdroege — APPROVED** [review]
Looks good to me otherwise. Can you replace that with a cast here (and in a separate commit for the others)?

**Comment by ids1024** [comment]
> Can you replace that with a cast here (and in a separate commit for the others)

Done.

**Review by bilelmoussaoui — APPROVED** [review]

**Pull request merged**